web/static/scripts/highlights.js

```diff
@@ -6,11 +6,22 @@ let DlCount = 0;
 
 for (let area of HIGHLIGHT_AREAS) {
     let code = area.querySelector("pre > code");
+    let name = area.querySelector(".pasteHeader > div > .filenameArea");
+
     pasteStores.push(code.textContent);
 
     // Highlight Code Block and get Language Details...
-    let details = hljs.highlightAuto(code.textContent);
-    let highlightedLang = details.language ? details.language : "plaintext";
+    let nameLang = getLangByName(name.textContent);
+    let highlightedLang;
+    let details;
+
+    if (!nameLang) {
+        details = hljs.highlightAuto(code.textContent);
+        highlightedLang = details.language ? details.language : "plaintext";
```

Comment:
We could avoid a ternary here by using the || OR
`highlightedLang = details.language || "plaintext";`